packages/store/package.json

```diff
@@ -26,6 +26,7 @@
     "ember-cli-babel": "^7.8.0",
     "ember-cli-path-utils": "^1.0.0",
     "ember-cli-typescript": "^2.0.2",
+    "ember-decorators-polyfill": "^1.0.6",
```

Comment:
I think we only need this in our ember-try config, I believe it would be on consuming apps to install the polyfill? Otherwise this lgtm.

cc @rwjblue

Reply:
If that is true, then we have to revert #6249 (since it means that ember-data@3.13 is not compatible with its own current support policy, and requires that the host app bring the polyfill).

🤔

Reply:
@rwjblue I’m more concerned with whether adding the polyfill would troll end consumers that bring their own version.

Reply:
I agree with @rwjblue, but lets discuss today

Reply:
fwiw it wouldn't mean we need to revert, only that we should change from using @computed to using a getter